### PR TITLE
【Hackathon No.6】为 Paddle 新增 trapezoid

### PR DESCRIPTION
**Pull request by kunkun0w0**
PaddlePaddle Hackathon 第三期 6

### CODE REVIEW

**Comment by Ligoml** [comment]
你好，需要按照要求修改pr标题，并发邮件发起评审哦~更多信息请见[活动详情](https://www.paddlepaddle.org.cn/contributionguide?docPath=hackathon_cn#huodongjieshao)
image

File: rfcs/APIs/20220708_api_design_for_trapezoid.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+- **dx** (float) - 相邻采样点之间的常数间隔；当**x**和**dx**均未指定时，**dx**默认为-1。
+  
+- **dim** (int) – 计算 trapezoidal rule 时 **y** 的维度。
+  
+
+输出是一个Tensor，其形状与 **y** 的形状与用于计算 trapezoidal rule 时的维度有关。
+
+## 底层OP设计
+
+使用已有API组合实现，不再单独设计OP。
```
Comment (jeff41404) [inline rfcs/APIs/20220708_api_design_for_trapezoid.md:202]: we need to list which APIs should be used to compose trapezoid, it is better to give some code demo.

File: rfcs/APIs/20220708_api_design_for_trapezoid.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+测试样例的构造思路：
+
+- 检查只输入 **y** 的情况
+  
+- 检查输入**y** 和 **dx** 的情况
+  
+- 检查输入 **y** 的 **x** 情况
+  
+
+以上3种测试样例中，分别考虑 **y** 为 1-D Tensor 和 **y** 为 n-D Tensor (n>1) 的情况
```
Comment (jeff41404) [inline rfcs/APIs/20220708_api_design_for_trapezoid.md:235]: we should consider more testing case, eg, check gradient case, some error case...

File: rfcs/APIs/20220708_api_design_for_trapezoid.md (hunk 1 above)
Comment (kunkun0w0) [reply rfcs/APIs/20220708_api_design_for_trapezoid.md]: finished

**Pull request merged**